draft-ietf-tls-tls13.md

```diff
@@ -1383,10 +1383,7 @@ Structure of this message:
 
 %%% Key Exchange Messages
 
-       struct {
-           uint8 major;
-           uint8 minor;
-       } ProtocolVersion;
+       uint8 ProtocolVersionID[2];
```

Comment:
0x0700 is also not a signature algorithm, but we don't call them SignatureSchemeIDs. We'd have to stick "ID" after every enum in the entire document.